packages/kbn-test/src/functional_tests/lib/run_kibana_server.js

```diff
@@ -38,7 +38,7 @@ export async function runKibanaServer({ procs, config, options }) {
       ...extendNodeOptions(installDir),
     },
     cwd: installDir || KIBANA_ROOT,
-    wait: /http server running/,
+    wait: /Kibana is now available/,
```

Comment:
IIRC you need to update here as well https://github.com/elastic/kibana/blob/master/x-pack/plugins/apm/e2e/run-e2e.sh#L150